pennylane_lightning/src/tests/Test_Measures.cpp

```diff
@@ -163,6 +163,55 @@ TEST_CASE("Expected Values", "[Measures]") {
     }
 }
 
+TEST_CASE("Sample", "[Measures]") {
```

Comment:
I wonder if we could add this as a templated test case for `float` and `double` types. I'd expect them to work fine, but not harm having it.

Something like:

```suggestion
TEMPLATE_TEST_CASE("Sample", "[Measures]", float, double) 
```


and using `TestType;` for the type throughout.

Reply:
Good idea!